aws/cloudformation/lambdas/sql-user/package.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
I like this approach of giving each lambda it's own package.json. It might slow down the build though, as each will have it's own node-modules folder. It also makes it harder to share testing resources or other common lambda helpers (like cfn-response?)

That said, for now it protects you from the old node versions being used elsewhere.